code/modules/projectiles/guns/mounted.dm

```diff
@@ -165,27 +165,33 @@
 
 	scatter = 10
 	deployed_scatter_change = -10
-	fire_delay = 0.5 SECONDS
+	fire_delay = 0.4 SECONDS
+	burst_amount = 12
```

Comment:
probs need a lower burst scatter mult, or just lower scatter in general. The burst scatter scales with burst amount, so its pretty high.